cli/cmd/connect.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/defenseunicorns/zarf/cli/internal/k8s"
+	"github.com/spf13/cobra"
+)
+
+var (
+	connectResourceName string
+	connectNamespace    string
+	connectResourceType string
+	connectLocalPort    int
+	connectRemotePort   int
+
+	connectCmd = &cobra.Command{
+		Use:   "connect <REGISTRY|LOGGING|GIT>",
+		Short: "Access services or pods deployed in the cluster.",
+		Run: func(cmd *cobra.Command, args []string) {
+			tunnel := k8s.NewTunnel(connectNamespace, connectResourceType, connectResourceName, connectLocalPort, connectRemotePort)
+			tunnel.Connect(args[0], true)
```

Comment:
If the args[0] isn't provided we hit a panic for index out of range. Can check if arguments have been provided and print the help if not.

Reply:
Looking at this more it looks like the  "REGISTRY", "LOGGING", "GIT" shortcuts are optional so instead of an arg this might serve better as a flag.